plugins/cloudservices/plugin.js

```diff
@@ -55,6 +55,13 @@
 			CloudServicesLoader.prototype.upload = function( url, additionalRequestParameters ) {
 				url = url || this.editor.config.cloudServices_url;
 
+				if ( !url ) {
+					CKEDITOR.error( 'cloudservices-url-error', {
```

Comment:
Following the convention from other plugins there should be no additional error message. It will be redundant as the additional info with link to docs with error description is always provided:

![image](https://user-images.githubusercontent.com/1061942/33014285-200863da-cde7-11e7-8198-5ceb5ef78640.png)

and we should include proper description there. Could you create new task in https://github.com/ckeditor/ckeditor-docs which will cover this?

Reply:
https://github.com/ckeditor/ckeditor-docs/issues/113